<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTableEdit</class>
 <widget class="QWidget" name="FormTableEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widgetList" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QWidget" name="widgetControls" native="true">
           <layout class="QVBoxLayout" name="verticalLayoutControls">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButtonCreate">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="DbUi.qrc">
                <normaloff>:/Icons/Menu New.png</normaloff>:/Icons/Menu New.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelActionsFirst">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/Icons/Separator.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonEdit">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="DbUi.qrc">
                <normaloff>:/Icons/Menu Edit.png</normaloff>:/Icons/Menu Edit.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonClone">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="DbUi.qrc">
                <normaloff>:/Icons/Menu Clone.png</normaloff>:/Icons/Menu Clone.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonExport">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="DbUi.qrc">
                <normaloff>:/Icons/Link.png</normaloff>:/Icons/Link.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonImport">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="DbUi.qrc">
                <normaloff>:/Icons/Unlink.png</normaloff>:/Icons/Unlink.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelActionsSecond">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/Icons/Separator.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonRemove">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="DbUi.qrc">
                <normaloff>:/Icons/Menu Remove.png</normaloff>:/Icons/Menu Remove.png</iconset>
              </property>
              <property name="autoRaise">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeView" name="treeViewTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="indentation">
          <number>6</number>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="headerShowSortIndicator" stdset="0">
          <bool>true</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widgetEdit" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widgetEditCtrl" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSave">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="DbUi.qrc">
             <normaloff>:/Icons/Menu Ok.png</normaloff>:/Icons/Menu Ok.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>357</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Menu New.png</normaloff>:/Icons/Menu New.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Menu Edit.png</normaloff>:/Icons/Menu Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Menu Remove.png</normaloff>:/Icons/Menu Remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Menu Clone.png</normaloff>:/Icons/Menu Clone.png</iconset>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Link.png</normaloff>:/Icons/Link.png</iconset>
   </property>
   <property name="text">
    <string>Export to .csv file</string>
   </property>
   <property name="toolTip">
    <string>Export to .csv file</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Unlink.png</normaloff>:/Icons/Unlink.png</iconset>
   </property>
   <property name="text">
    <string>Import from .csv file</string>
   </property>
   <property name="iconText">
    <string>Import from .csv file</string>
   </property>
   <property name="toolTip">
    <string>Import from .csv file</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="DbUi.qrc">
     <normaloff>:/Icons/Refresh.png</normaloff>:/Icons/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>toolButtonCreate</tabstop>
  <tabstop>toolButtonEdit</tabstop>
  <tabstop>toolButtonClone</tabstop>
  <tabstop>toolButtonExport</tabstop>
  <tabstop>toolButtonImport</tabstop>
  <tabstop>toolButtonRemove</tabstop>
  <tabstop>treeViewTable</tabstop>
 </tabstops>
 <resources>
  <include location="DbUi.qrc"/>
 </resources>
 <connections/>
</ui>
